Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -3319,7 +3319,12 @@ void WebPageProxy::performDragOperation(DragData& dragData, const String& dragSt
     if (!hasRunningProcess())
         return;
 
-    sendWithAsyncReply(Messages::WebPage::PerformDragOperation(dragData, WTFMove(sandboxExtensionHandle), WTFMove(sandboxExtensionsForUpload)), [this, protectedThis = Ref { *this }] (bool handled) {
+    sendToProcessContainingFrame(frameID, Messages::WebPage::PerformDragOperation(frameID, dragData, WTFMove(sandboxExtensionHandle), WTFMove(sandboxExtensionsForUpload)), [this, protectedThis = Ref { *this }, dragData, dragStorageName] (bool handled, std::optional<WebCore::RemoteUserInputEventData> remoteUserInputEventData, std::optional<SandboxExtensionHandle> sandboxExtensionHandle, std::optional<Vector<SandboxExtension::Handle>> sandboxExtensionsForUpload) mutable {
```

Comment:
It seems unnecessary to pass the same sandbox extensions back through the completion handler. Is there a way we could capture the sandbox extensions in the lambda instead?